pages/profiles/snigdharao.md

```diff
@@ -6,10 +6,19 @@
 ### BuzzWords :Python, Java, C/C++, JS, Bash, MIPS, Matlab, Linux, Postgres, MySQL, AWS
 ***
 As a recent graduate with a Master’s in Computer Science from California State University, Long Beach,I am passionate about coding and love to learn new tech stacks .I have  over 2 years of work experience and i love to travel
-# Projects
+---
```

Comment:
@snigdharao55 By adding an underline here, it makes the whole paragraph above it into a H2 header. Is that what you want?

Reply:
No , I will fix it now .

Reply:
Yes , i have added h2(##) for it .Do i have to add any more tags?